src/openstudio_app/CMakeLists.txt

```diff
@@ -224,6 +234,10 @@ elseif( WIN32 )
     COMMAND ${CMAKE_COMMAND} -E copy_if_different ${qweb_resources_100} $<TARGET_FILE_DIR:${target_name}>/
     COMMAND ${CMAKE_COMMAND} -E copy_if_different ${qweb_resources_200} $<TARGET_FILE_DIR:${target_name}>/ )
 
+  add_custom_command(TARGET ${target_name}
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:openstudio::openstudiolib> $<TARGET_FILE_DIR:${target_name}>/ )
+
```

Comment:
Without this, when launching on Windows from my local build dir: build_dir/Products/Release/OpenStudioApp.exe

![image](https://user-images.githubusercontent.com/5479063/78614254-5e2d8a80-786e-11ea-920c-06dcb8c87f5c.png)


@kbenne would like your review here.

Reply:
It might be a wrongly calculated `_IMPORT_PREFIX` or something on windows. Copying shouldn't be necessary (but works)